backend/data/schema/Member.js

```diff
@@ -130,7 +130,6 @@ const MemberSchema = new Schema({
     },
     tokenExpiry: {
         type: Number,
```

Comment:
tokenExpiry field is being returned by default when querying this MongoDB schema. This just makes things easier.